api-practice/newpaper.ipynb

```
@@ -19,9 +19,200 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
```

Ruff (0.13.1)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)